tracee/tracee.go

```diff
@@ -429,10 +435,18 @@ func (t *Tracee) populateBPFMaps() error {
 		fileFilterMap.Update(uint32(i), []byte(t.config.FilterFileWrite[i]))
 	}
 
-	uidHash, _ := t.bpfModule.GetMap("uid_filter")
-	for i := 0; i < len(t.config.Filter.UIDs); i++ {
-		uidHash.Update(uint32(t.config.Filter.UIDs[i]), uint32(1))
+	// Set UID filter options
+	uidEqualityFilter, _ := t.bpfModule.GetMap("uid_equal_filter")
```

Comment:
I don't have the error checked here because it's not checked elsewhere in the file, but i'm curious why that is?

Reply:
TL;DR - we should add those checks

When I wrote the libbpfgo, I wanted to keep the interface as close as possible to what we had with gobpf,
so it will be easy to understand the commit.
As gobpf doesn't return an error when using it's NewTable method (not sure about the name), I didn't handle it here. However, libbpfgo does return an error (for example, if the map doesn't exist).
This error should never happen with tracee, as we know these maps do exist in our bpf code, but it is always a good practice to check for errors